tests/unit/ext/switches/brocade.py

```diff
@@ -205,15 +207,17 @@ def test_get_mode(self, switch):
             response = switch._get_mode(INTERFACE1)
             assert response == 'trunk'
 
-    def test_apply_networking(self, switch, nic, network):
+    def test_modify_port(self, switch, nic, network):
         # Create a port on the switch and connect it to the nic
         port = model.Port(label=INTERFACE1, switch=switch)
-        nic.port = port
+        port.nic = nic
```

Comment:
This is simply because in our defined model (see model.py), class `Port` is defined with a field `nic` but not vice versa. Calling `nic.port` is not a relation we captured in our database.